packages/mermaid/src/types.ts

```diff
@@ -1,6 +1,12 @@
 export interface NodeMetaData {
   shape?: string;
   label?: string;
+  icon?: string;
+  form?: string;
+  pos?: 't' | 'b';
+  img?: string;
+  w?: string;
+  h?: string;
```

Comment:
```suggestion
  width?: string;
  height?: string;
```

Reply:
Short names are spl request from @knsv

Reply:
We're using `width, height` everywhere else. And because we're a library with downstream customers, I, personally would prefer explicit names.  It'll also make reading the code easier for new contributors. 

But upto @knsv